### PR TITLE
Update LIT2393tabiba.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
update

### CODE REVIEW

**Comment by smaugustine** [comment]
Just to note, the former IDs are LIT3084RepCh364 and LIT3092RepCh372 (the first numeric parts are missing in the commit).

**Comment by DenisNosnitsin1970** [comment]
Indeed. Thanks!

**Comment by thea-m** [comment]
Thank you! Will you make a new PR to delete the other records? Have references to the old IDs the manuscripts been updated?

**Comment by DenisNosnitsin1970** [comment]
Not yet, I will do that.

**Comment by thea-m** [comment]
Then I will merge this when the manuscript PR is done, otherwise we will have chaos

File: 2001-3000/LIT2393tabiba.xml [hunk 1]
```diff
@@ -51,8 +51,40 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+          <ab>
+          <l n="1">ኦፈጣሬ፡ ዓለማት፡ በኀይለ፡ አርምሞ፡ ወነቢብ፡ </l>
+          <l n="2">እምጥንተ፡ ቀዳማይ፡ እሑድ፡ እስከ፡ ሰርከ፡ ስዱስ፡ ዐርብ፡ </l>
+          <l n="3">እግዚአብሔር፡ ኀያል፡ እግዚአብሔር፡ ጠቢብ፡ </l>
+          <l n="4">ቀደሱከ፡ ደርገ፡ አሜሃ፡ ትጉሃነ፡ ሰማይ፡ ወኪሩብ፡ </l>
+            <l n="5">እስመ፡ እንበለ፡ አብ፡ ተረክበ፡ ባሕቲትከ፡ አብ። </l>
+            </ab>
+          </div>
+    </div>
       </body>
```
Comment (eu-genia) [inline 2001-3000/LIT2393tabiba.xml:82]: The bibliography from the deleted records is missing and must be provided here
```
<listBibl type="editions">
               <bibl>
                  <ptr target="bm:Chaine1913Rep"/>
                  <citedRange unit="item">364</citedRange>
               </bibl>
 <bibl>
                  <ptr target="bm:Chaine1913Rep"/>
                  <citedRange unit="item">372</citedRange>
               </bibl>
</listBibl>
```

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Comment by eu-genia** [comment]
Just to link this to the related PR in MSS: https://github.com/BetaMasaheft/Manuscripts/pull/796

File: 2001-3000/LIT2393tabiba.xml [hunk 2]
```diff
@@ -56,6 +56,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <citedRange unit="page">118-163</citedRange>
                </bibl>
             </listBibl>
+            <div type="bibliography">
```
Comment (eu-genia) [inline 2001-3000/LIT2393tabiba.xml:59]: there is something wrong with the nesting, you already had the `<div type="bibliography">` opened in line 48 above, please remove line 59

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Review by eu-genia — APPROVED** [review]

**Pull request merged**